src/Tickets/Seating/Service/Layouts.php

```diff
@@ -125,7 +128,40 @@ public function get_in_option_format() {
 
 		return $layouts;
 	}
-
+	
+	/**
+	 * Fetches all the Layouts from the database.
+	 *
+	 * @since TBD
+	 *
+	 * @return Layout_Card[] Array of layout card objects.
+	 */
+	public function get_in_card_format() {
+		if ( ! $this->update() ) {
+			return [];
+		}
+		
+		$mem_key      = 'option_layout_card_objects';
+		$layout_cards = tribe_get_var( $mem_key );
+		
+		if ( ! ( $layout_cards && is_array( $layout_cards ) ) ) {
+			$layout_cards = [];
+			foreach ( Layouts_Table::fetch_all() as $row ) {
+				$layout_cards[] = new Layout_Card(
+					$row->id,
+					$row->name,
+					$row->map,
+					$row->seats,
+					$row->screenshot_url
+				);
+			}
+			
+			tribe_set_var( $mem_key, $layout_cards );
```

Comment:
To memoize use `tribe_cahe()[$mem_key]`. That will go into non-persistent caching in WP cache layer and will be naturally flushed with a `wp_cache_flush` call.
The same is not true for `tribe_set_var()`; the var will be set for the whole request (or a whole test process run)